runtime/pangolin/Cargo.toml

```diff
@@ -319,11 +319,12 @@ runtime-benchmarks = [
 	"pallet-collective/runtime-benchmarks",
 	"pallet-democracy/runtime-benchmarks",
 	"pallet-elections-phragmen/runtime-benchmarks",
-	"pallet-membership/runtime-benchmarks",
 	"pallet-identity/runtime-benchmarks",
+	"pallet-membership/runtime-benchmarks",
 	"pallet-preimage/runtime-benchmarks",
 	"pallet-proxy/runtime-benchmarks",
 	"pallet-scheduler/runtime-benchmarks",
+	"pallet-sudo/runtime-benchmarks",
```

Comment:
Reuse upstream weight info, this line can be deleted.

Reply:
Can you elaborate on the "satisfaction" here?  CI requires to enable all pallet's benchmark feature?